docker-compose-generator/docker-fragments/bitcoin-eclair.yml

```diff
@@ -0,0 +1,55 @@
+version: "3"
+
+services:
+  bitcoind:
+    environment:
+      BITCOIN_NETWORK: ${NBITCOIN_NETWORK:-regtest}
+      BITCOIN_EXTRA_ARGS: |
+        zmqpubrawblock=tcp://0.0.0.0:28332
+        zmqpubrawtx=tcp://0.0.0.0:28333
+        deprecatedrpc=signrawtransaction
+        addresstype=p2sh-segwit
```

Comment:
do we really need this?

Reply:
Eclair needs it

Reply:
Line `deprecatedrpc=signrawtransaction` is not necessary anymore, since 0.3 eclair uses `signrawtransactionwithwallet`.

Reply:
@araspitzu and about `addresstype`?

Reply:
It can be removed as well, eclair supports both `p2sh-segwit` and `bech32` addresses and since bitcoin 0.16 the default is `p2sh-segwit` so this line can be safely removed.